config/kubernetes/sig-node/teams.yaml

```diff
@@ -9,6 +9,12 @@ teams:
     - thockin
     - yujuhong
     privacy: closed
+  sig-node-leads:
+    description: Chairs and Technical Leads for SIG Node
+    members:
+    - dchen1107
+    - derekwaynecarr
```

Comment:
```suggestion
    - derekwaynecarr
    - mrunalp
    - SergeyKanzhelev
```

Reply:
+1 lets align this with the new structure while we are here.

Reply:
should I also update the OWNERS_ALIAS file in this PR than?

Reply:
Yeah, why not?